packages/documentation/src/pages/getting-started/common-tasks/new-unit-test.mdx

```diff
@@ -239,24 +292,18 @@ mockFetch(
   false, // shouldResolve: false returns rejected promise. default is true
 );
 ```
-
 **Note:** This utility can be found in `platform/utilities/storage/localstorage.js`.
-
 - `getLocalStorage()` - convenience accessor for local storage and implements a fallback. Useful for stubbing localstorage.
 
-
 ## Legacy tools
-
-While ReactTestUtils and SkinDeep are used in many of our tests, use Enzyme for any updated or new tests.
-
+While ReactTestUtils, SkinDeep, and Enzyme are used in many of our tests, use React Testing Library for any updated or new tests.
 ## Libraries
-
 Unit tests use the following libraries:
-
 - [mocha.js](https://mochajs.org/): Test framework.
 - [chai.js](https://www.chaijs.com/): BDD / TDD assertion library.
 - [chai-as-promised](https://github.com/domenic/chai-as-promised): Extends Chai with assertions about promises.
 - [sinon.js](https://sinonjs.org/): Standalone test spies, stubs and mocks.
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro): React Testing Library is a branch of `Testing Library` that allows users to test their components through actual DOM nodes instead of rendered instances.
```

Comment:
```suggestion
- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro): React Testing Library is a branch of `Testing Library` that allows users to test their React components through actual DOM nodes instead of rendered instances.
```

Reply:
- [x] Specify "React components" instead of just general "components"